### PR TITLE
1953 implementation

### LINKED ISSUE #1953: Email Template URLs
**Describe the bug**
When a user edits an email template Summernote and/or the browser can mangle the URL beyond comprehension.

**Janeway version**
1.2.9

**To Reproduce**
1. Try to paste a default template value into a summernote box

**Expected behavior**
URLs should be rendered in code and passed through via the context so the placeholder will look like `{{ a_url }}`.

### LINKED ISSUE #1963: Email Template: review_complete_reviewer_acknowledgement
**Describe the bug**
This email is signed off by the current user rather than the Editor. 

**Janeway version**
1.3.9

### PR DESCRIPTION
**Pull request by ajrbyers**
Closes #1953 
Closes #1963

### CODE REVIEW

File: src/utils/install/journal_defaults.json [hunk 1]
```diff
@@ -239,21 +239,6 @@
...
-            "description": "Email sent when a review assignment is updated.",
-            "is_translatable": true,
-            "name": "review_request_sent",
-            "pretty_name": "Review Request Update",
-            "type": "rich-text"
-        },
-        "value": {
-            "default": "Dear {{ review_assignment.editor.full_name }}, <br><br>This is a notification that the reviewer status on \"{{ article.title }}\" in {{ article.journal.name }} has been <a href=\"{% journal_url 'review_in_review' review_assignment.article.pk  %}\">updated</a>. <br><br>Regards, <br>{{ request.user.signature|safe }}"
-        }
-    },
```
Comment (mauromsl) [inline src/utils/install/journal_defaults.json:256]: Looks like an accidental deletion

File: src/utils/install/journal_defaults.json [hunk 2]
```diff
@@ -2755,6 +2864,21 @@
...
+            "description": "Changes the generic text above the File Upload box on the Peer Review form page.",
+            "is_translatable": true,
+            "name": "peer_review_upload_text",
+            "pretty_name": "Peer Review Upload Text",
+            "type": "rich-text"
+        },
+        "value": {
+            "default": "<p>You may upload a file in lieu of completing the review form. Please ensure you've covered all of the required fields below.</p>"
+        }
+    },
```
Comment (mauromsl) [inline src/utils/install/journal_defaults.json:2881]: I think this will lead to a dupe

File: src/utils/install/journal_defaults.json [hunk 3]
```diff
@@ -2770,4 +2894,4 @@
             "default": "<p>You may upload a file in lieu of completing the review form. Please ensure you've covered all of the required fields below.</p>"
         }
     }
-]
+]
```
Comment (mauromsl) [inline src/utils/install/journal_defaults.json:2897]: Worth re-instating this EOF

File: src/core/migrations/0046_delete_review_request_sent.py [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    )
+
+    for setting in prod_complete_settings:
+        setting.value = setting.value.replace(
+            '<p>This article is now in the Proofing workflow: {{ proofing_list_url }}.</p>',
+            '',
+        )
+        setting.value = setting.value.replace(
+            'Dear {{ assignment.editor.full_name }},</p><p>This is an automatic',
+            'This is a',
```
Comment (mauromsl) [inline src/core/migrations/0046_delete_review_request_sent.py:31]: It seems this didn't work/get applied, the new defaults don't match this replace

File: src/core/migrations/0046_delete_review_request_sent.py [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+    notify_editor_proofing_complete_settings = SettingValueTranslation.objects.filter(
+        master__setting__name='notify_editor_proofing_complete',
+        master__setting__group__name='email',
+    )
+
+    for setting in notify_editor_proofing_complete_settings:
+        setting.value = setting.value.replace(
+            ': {{ publish_url }}',
+            '',
```
Comment (mauromsl) [inline src/core/migrations/0046_delete_review_request_sent.py:55]: This change is not reflected in the defaults

**Review by mauromsl — CHANGES_REQUESTED** [review]
There are some inconsistencies on the new JSON defaults that don't match the migrations

File: src/core/migrations/0046_delete_review_request_sent.py [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+        master__setting__name='review_complete_reviewer_acknowledgement',
+        master__setting__group__name='email',
+    )
+
+    for setting in review_ack_settings:
+        setting.value = setting.value.replace(
+            'request.user.signature',
+            'review_assignment.editor.signature',
+        )
+        setting.save()
```
Comment (mauromsl) [inline src/core/migrations/0046_delete_review_request_sent.py:69]: This is not reflected on the JSON defaults

File: src/core/migrations/0046_delete_review_request_sent.py (hunk 5 above)
Comment (ajrbyers) [reply src/core/migrations/0046_delete_review_request_sent.py:55]: Default is:

```
<p>Dear {{ proofing_assignment.editor.full_name }},</p><p>Proofing for "{{ article.title }}" is now complete.</p><p>Regards,</p><p>{{ request.user.signature|safe }}</p>
```
{{ publish_url }} has been removed as far as I can tell.

Comment (mauromsl) [reply src/core/migrations/0046_delete_review_request_sent.py:55]: Why do we need a migration to delete it then? Is it present on some installs?

Comment (ajrbyers) [reply src/core/migrations/0046_delete_review_request_sent.py:55]: Yes - it certainly was in mine.

**Comment by ajrbyers** [comment]
**Notes from Testing**

- Proofing
    - Correction request email should go to /proofing/requests/corrections/ not /proofing/requests/
    - When undertaking corrections the View Metadata will post the form
    - Proofing complete email sends you to Production rather than Proofing
- Typesetting Plugin
    - When proofing complete email is sent add link to Typesetting article page

**Pull request merged**